web/themes/custom/votegov/templates/component/page-title.html.twig

```diff
@@ -18,8 +18,6 @@
 
 {{ title_prefix }}
 {% if title %}
-  <div class="page-title">
-    <h1{{ title_attributes.addClass('main-heading').setAttribute('data-test', 'mainHeader') }}>{{ title }}</h1>
-  </div>
+    <h1{{ title_attributes.addClass('page-title').setAttribute('data-test', 'mainHeader') }}>{{ title }}</h1>
```

Comment:
Removed div since there was just the `h1` and removed the `mainHeader` class... i did not find it being used any where else